.github/workflows/rosa-scaling-benchmark.yml

```diff
@@ -453,36 +452,3 @@ jobs:
         with:
           region: ${{ inputs.region }}
 
-  archive:
-    name: Commit results to Git repository
-    runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      actions: write
-    needs:
-      - run
-    steps:
-      - name: Checkout repository for results
-        uses: actions/checkout@v4
-        with:
-          ref: 'result-data'
-
-      - uses: actions/download-artifact@v4
-        with:
-          name: final-report-json-${{ inputs.outputArchiveSuffix }}
-
-      - name: Commit result-summary
-        shell: bash
-        env:
-          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git config --global user.name "github-actions[bot]"
-          DATE_FOLDER=rosa_scalability/$(date +"%Y/%m/%d")
-          mkdir -p ${DATE_FOLDER}
-          mv *.json ${DATE_FOLDER}
-          git add .
-          git commit -m "generated"
-          git push
-
```

Comment:
I'm not 100% sure whether it's ok to remove this but it's probably better fot the older branches like 0.14-relase not to pollute latest perf results with old releases.